package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-http-auth",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
Frankly I'm not very familiar with `-SNAPSHOT` stuff.

The way how I've seen it being done (and how I do it), is that once you decide to publish a feature, you push version bumps+changelog to the master and tag a new version at Github to that commit, so that the head of new version is always basically just a version number change. There are also gulp/bash/rake/grunt scripts to do this for you.

You wouldn't want version number changes to package.json within PRs because there might be multiple PR's going on at the same time and once one of them gets merged, you might want to publish new version before merging the next one etc... it gets into a mess to manage quickly. :-)

Similarly  you don't want to have code-style changes, new features, multiple fixes etc within one code. It gets harder for others what actually has been changed. _One thing per PR._

See also [Semver](http://semver.org) to make sure you do proper versioning regarding breaking changes (don't be afraid to bump up the first digit, basically).

Hope this helps! :-)